### ISSUE TITLE
Citing the library?

### ISSUE THREAD

**Issue by NicklasXYZ** [ThomasBury/arfs#5]
Hello Thomas

Thanks for a great library!

I am getting good results with the GrootCV procedure and was wondering if you have any work published on this algorithm?
I am interested in using it in a study and it would be great to have something to cite :)
Otherwise how would you like the library to be cited?

**Comment by ThomasBury** [comment]
> Hello Thomas
> 
> Thanks for a great library!
> 
> I am getting good results with the GrootCV procedure and was wondering if you have any work published on this algorithm? I am interested in using it in a study and it would be great to have something to cite :) Otherwise how would you like the library to be cited?

Hi @NicklasXYZ,

Sorry for the delay. I'm glad you found ARFS useful 🙂. No peer-reviewed publication yet. You can cite using the "cite this repository" button on the main page ;)

I supervised one "project thesis" (@kottorov) available [here](https://github.com/kottorov/arfs-tests). I'm currently supervising a second master thesis (the defense is mid-Jan). I don't know yet if it will be made publicly available or not.

In summary, the students studied the effect of collinearity and the performance impact it has, in different ways. For any feature selection methods I know, ARFS or Minimal-Optimal, collinearity is problematic. The goal was to quantify and study the robustness. This was initiated on the [naive collinearity example NB](https://github.com/ThomasBury/arfs/blob/main/examples/Collinearity.ipynb) I wrote at the time.

**Comment by NicklasXYZ** [comment]
Thanks a lot for the reply and the additional information you provided!

I ended up doing things in a slightly different way, but might re-visit the project again in the near future to try it out with arfs :)
